src/java/org/apache/cassandra/repair/messages/ValidationRequest.java

```diff
@@ -61,27 +63,29 @@ public boolean equals(Object o)
     @Override
     public int hashCode()
     {
-        return nowInSec;
+        // TODO 14227 better hash possible?
```

Comment:
Doesn't seem worse than the previous hash

Reply:
Thx.